keyboards/hackpad/info.json

```
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Nico Stuhlmueller",
+    "keyboard_name": "hackpad",
+    "maintainer": "ThePurox",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder" : true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B3", "B2", "B6"],
+        "rows": ["C6", "D7", "E6", "B4", "B5"]
+    },
+    "encoder" : {
+        "rotary" : [
+            {"pin_a" : "F5", "pin_b" : "F4"}
+        ]
+    },
+    "processor": "atmega32u4",
+    "url": "https://git.imaginaerraum.de/Purox/Hackpad",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "rgblight" : {
+        "led_count" : 16,
+        "pin": "D4",
```

```suggestion
    "ws2812": {
        "pin": "D4"
    },
    "rgblight" : {
        "led_count" : 16,
```
Your PR targets the `develop` branch, where `rgblight.pin` has been replaced by `ws2812.pin` (#20303).